be/src/olap/utils.cpp

```diff
@@ -1432,9 +1451,10 @@ OLAPStatus dir_walk(const string& root,
     struct dirent* direntp = NULL;
     dirp = opendir(root.c_str());
     if (dirp == nullptr) {
-        NewStatus status = IOError(strings::Substitute("opendir $0 failed", root), errno);
-        LOG(WARNING) << status.ToString();
-        if (status.IsDiskFailure()) {
+        Status status = disk_error("opendir failed", errno);
```

Comment:
1. After you replace NewStatus, it is not used by anyone. You should remove these files and modify NOTICE too.
2. You can reimplement this function with interface in env.

Reply:
Yeah.
1. Will remove the new_status file
2. I will refactor the method in other issused